subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r40/BuildCacheCleanupOperationsCrossVersionSpec.groovy

```diff
@@ -72,11 +83,14 @@ class BuildCacheCleanupOperationsCrossVersionSpec extends ToolingApiSpecificatio
             }
         }
         then:
-        cacheDir.directorySize() >= 4*1024*1024
+        assert cacheDir.directorySize() >= 4 * 1024 * 1024
```

Comment:
The assert is superflous.
Do you also want to reformat the [multiplication further down](https://github.com/gradle/gradle/pull/3856/files#diff-e005e675e3def6a962318a92e8969defR107)?

Reply:
OK. Not sure where the link points, can you give me line numbers?

Reply:
Fixed.